_1327/minutes/templates/minutes_list.html

```diff
@@ -63,15 +64,16 @@ <h3 id="year{{ year }}">{{ year }}</h3>
 				</tr>
 			{% endfor %}
 		</table>
-    {% empty %}
-    	{% block minutesempty %}
-	        <em>{% trans "No minutes available." %}</em>
-	        {% url "login" as anchor_url %}
-	        {% if not user.is_authenticated %}
-	            <em>
-	                {% blocktrans with anchor='<a href="'|add:anchor_url|add:'">'|safe anchor_end='</a>'|safe %}You might have to {{ anchor }} login {{ anchor_end }} first.{% endblocktrans %}
-	            </em>
-	        {% endif %}
-	    {% endblock %}
+	{% empty %}
+		{% block minutesempty %}
+			<em>{% trans "No minutes available." %}</em>
+			{% url "login" as anchor_url %}
+			{% redirect_login as redirect_params %}
```

Comment:
why can't you use it directly?

Reply:
I think this would break the translation, wouldn't it?

Reply:
`{% blocktrans with anchor='<a href="'|add:anchor_url|add:redirect_login|add:'">'|safe anchor_end='</a>'|safe %}...{% endblocktrans %}` should work fine i think.

Reply:
I think the problem is that we can not nest multiple template filters and the template system does not necessarily know that it should render the filter,but it would make sense to try it